docs/UserGuide.md

```diff
@@ -407,6 +484,32 @@ If your changes to the data file makes its format invalid, MyInsuRec will discar
 
 --------------------------------------------------------------------------------------------------------------------
 
-## 6. Glossary
+## 8. Troubleshooting
+
+This section covers technical issues you may run into while using *MyInsuRec*.
+
+### 8.1 Checking your system's Java version
+
+To check that your system has the correct Java version (Java 11 and above) to run *MyInsuRec*, you can follow the steps below:
+
+1. Open your terminal.
+   * Windows
+     * Use `Win` + `S` to open search.
+     * Type in 'Terminal' to search for it and click on it to launch.
+   * macOS
+     * Use `Cmd` + `Space` to open Spotlight search.
+     * Type in 'Terminal' to search for it and click on it to launch.
```

Comment:
I think u can also add in for linux machine

For them to check, it just use 
`ctr+alt+T` this will launch the terminal straight away.